examples/data_collect/main.cpp

```diff
@@ -222,7 +220,13 @@ int main(int argc, char *argv[]) {
     n_frames = std::stoi(command_map[{"-n", "--n"}]);
 
     // Parsing mode type
-    mode = std::stoi(command_map[{"-m", "--m"}]);
+    std::string modeName;
+    try {
+        mode = std::stoi(command_map[{"-m", "--m"}]);
```

Comment:
I believe std::stoi() will succeed if the string to convert is "1a" for example. It will set mode to value 1 (integer).
Please use the 2nd parameter of std::stoi() as help to validate the "1a" situation.

Reply:
Case where -m has both int and string present is solved.